jcl/src/java.base/share/classes/com/ibm/oti/vm/VM.java

```diff
@@ -262,12 +264,29 @@ private VM() {
  * @param loaderType
  *            J9_CLASSLOADER_TYPE_BOOT     - bootstrap classloader
  *            J9_CLASSLOADER_TYPE_PLATFORM - platform classloader
- *            J9_CLASSLOADER_TYPE_OTHERS   - other classloader
+ *            J9_CLASSLOADER_TYPE_APP      - application classloader
+ *            J9_CLASSLOADER_TYPE_OTHERS   - other classloaders. Also used for the initialization of the
+ *                                           application classloader if Java 8 or RAM_CLASS_CACHE_SUPPORT is
+ *                                           not enabled.
```

Comment:
Remove: Also used for the initialization of the
 *                                           application classloader if Java 8 or RAM_CLASS_CACHE_SUPPORT is
 *                                           not enabled.

Reply:
As things are, any time the app loader is use it will be used with the `J9_CLASSLOADER_TYPE_APP` ID.